src/design/optForce/optForce.m

```diff
@@ -0,0 +1,1000 @@
+function [optForceSets, posOptForceSets, typeRegOptForceSets, flux_optForceSets] = optForce(model,...
+    targetRxn, mustU, mustL, minFluxesW, maxFluxesW, minFluxesM, maxFluxesM, k,...
+    nSets, constrOpt, excludedRxns, runID, outputFolder, outputFileName,... 
+    printExcel, printText, printReport, keepInputs, verbose)
+% This function runs the third step of optForce that is to solve a
+% bilevel mixed integer linear programming problem to find sets of
+% interventions that lead to an increased production of a particular target
+%
+%
+% USAGE: 
+%
```

Comment:
@snmendoz, we have to merge PR #498 first. I only solved that conflict yesterday afternoon.

In order to speed things up, try to make smaller PRs. For instance, PR #498 has close to 1 million lines(!) to be added. Also, if you could, try to publish to the same feature (branch) when modifying the same files, without duplicating code in another PR or submitting another PR with similar files when a PR is already open. It makes it easier for us to review the contribution. Thanks.

*(Note to myself: provide these as guidelines on how to efficiently use the MATLAB.devTools.)*